src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import core.basesyntax.figure.Figure;
+import core.basesyntax.supplier.ColorSupplier;
+import core.basesyntax.supplier.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier(new ColorSupplier());
+        Figure[] figures = {figureSupplier.getRandomFigure(),
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getDefaultFigure(),
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getDefaultFigure(),
+                figureSupplier.getRandomFigure()};
+        for (Figure figure : figures) {
+            figure.draw();
+        }
+
```

Comment:
remove empty line
generate figures in loop
what if we wanted to generate 100 figures?